.circleci/config.yml

```diff
@@ -67,15 +62,15 @@ jobs:
         environment:
           PYTHON_VERSION: py27
 
-  build-dash-37: &build-dash
+  build-dash-39: &build-dash
     working_directory: ~/project
```

Comment:
not sure if it has been discussed before but I observed some dependency issues locally when using 3.9. We might want to consider both keeping 3.7 (or 3.8) and 3.9 for a while.

Reply:
As discussed on slack, we're still testing in 3.6, so between that an 3.9 there should be very little additional value in a 3.7 test.